docs/src/test-api/class-workerinfo.md

```diff
@@ -2,23 +2,8 @@
 * since: v1.10
 * langs: js
 
-`WorkerInfo` contains information about the worker that is running tests. It is available to [`method: Test.beforeAll`] and [`method: Test.afterAll`] hooks and worker-scoped fixtures.
+`WorkerInfo` contains information about the worker that is running tests, available to the worker-scoped fixtures. `WorkerInfo` is a subset of [TestInfo] that is available in many other places.
```

Comment:
`WorkerInfo` contains information about the worker that is running tests and is available to worker-scoped fixtures. `WorkerInfo` is a subset of [TestInfo] that is available in many other places.